app/lib/methods/helpers/buildMessage.ts

```diff
@@ -2,7 +2,7 @@ import { IMessage } from '../../../definitions';
 import messagesStatus from '../../../constants/messagesStatus';
 import normalizeMessage from './normalizeMessage';
 
-export default (message: IMessage): IMessage => {
+export default (message: IMessage): IMessage | null => {
```

Comment:
Why could the return be null when we always pass the same object?

Reply:
Because of `normalizeMessage`, which returns null if message is falsy.
```ts
export default msg => {
	if (!msg) {
		return null;
	}
```